autobump-maven/build.gradle

```diff
@@ -6,6 +6,7 @@ dependencies {
 
     implementation group: 'org.apache.maven', name: 'maven-model', version: '3.6.3'
     implementation group: 'org.apache.maven', name: 'maven-project', version: '2.2.1'
+    implementation group: 'org.apache.maven.artifact', name: 'maven-artifact', version: '3.0-alpha-1'
```

Comment:
I think this is an old version of the `maven-artifact` dependency. I think the new one uses group `org.apache.maven`, name: `maven-artifact` and version: `3.6.3`.
It might be better to use the latest version instead of an old alpha version.